tests/integration/folders/test_set_folder_status.py

```diff
@@ -0,0 +1,89 @@
+from unittest import TestCase
```

Comment:
It will be better to have an assertion on the number of log lines

Reply:
done